modules/commands/list.json

```diff
@@ -520,6 +520,24 @@
       }
     ]
   },
+  {
+    "name": "tab",
+    "aliases": [
+      "tablist"
+    ],
+    "title": "Tablist",
+    "description": "LuckPerms is only able to send information to formatting plugins so that they display them properly. LuckPerms does not put the prefix in front of your name on the tablist, you need a tab formatting plugin to do it, and you need Vault. Same goes for the chat! If the information appears correctly when you run `/lp user <user> info`, LuckPerms is doing its job!",

```

Comment:
It is not necessarily the case that you need Vault to display a prefix in tab, as many tab plugins simply use PAPI